kickoff/jsonexport.py

```diff
@@ -0,0 +1,244 @@
+
+
+@app.route('/json/firefox_history_stability_releases.json', methods=['GET'])
+def firefox_history_stability_releases_json():
+    # Match X.Y.Z (including esr) + W.X.Y.Z (example 1.5.0.8)
+    values = getFilteredReleases("firefox", "stability")
+    return jsonify(values)
+
+
+@app.route('/json/firefox_history_development_releases.json', methods=['GET'])
+def firefox_history_development_releases_json():
+    # Match 23.b2, 1.0rc2, 3.6.3plugin1 or 3.6.4build7
+    values = getFilteredReleases("firefox", "dev")
+    return jsonify(values)
+
+
+@app.route('/json/firefox_versions.json', methods=['GET'])
+def firefox_versions_json():
+    # Stable
+    lastStable = getFilteredReleases("firefox", ["major", "stability"], lastRelease=True)
+    firefoxversions["LATEST_FIREFOX_VERSION"] = lastStable[0][0]
+    # beta
+    lastStable = getFilteredReleases("firefox", ["dev"], lastRelease=True)
+    firefoxversions["LATEST_FIREFOX_DEVEL_VERSION"] = lastStable[0][0]
+    firefoxversions["LATEST_FIREFOX_RELEASED_DEVEL_VERSION"] = lastStable[0][0]
+    # esr
+    lastStable = getFilteredReleases("firefox", ["esr"], lastRelease=True)
+    firefoxversions["FIREFOX_ESR"] = lastStable[0][0]+"esr"
+    # esr next
+    lastStable = getFilteredReleases("firefox", ["esr"], lastRelease=True, ESR_NEXT=True)
+    if lastStable:
+        # If not found, that means that we are managing only a single ESR
+        firefoxversions["FIREFOX_ESR_NEXT"] = lastStable[0][0]+"esr"
+    # aurora
+    firefoxversions["FIREFOX_AURORA"] = config.AURORA_VERSION
+    # old version
+    firefoxversions["LATEST_FIREFOX_OLDER_VERSION"] = config.LATEST_FIREFOX_OLDER_VERSION
+    return jsonify(firefoxversions)
+
+
+def replaceValue(array, versionTag, version):
+    for locale, datalist in array.iteritems():
+        i = 0
+        for ver in datalist:
+            if ver == versionTag:
+                array[locale][i] = version
+            i += 1
+
+
+def generateLocateBuilds(buildsVersionLocales, l10nchangesets, lastVersion):
+    # The l10n format for desktop is 'locale changeset'
+    # parse it
+    for locale in l10nchangesets.split('\n'):
+        if "\t" in locale:
+            # Not sure why, but in some cases, \t is used a separator
+            localeCode = locale.split("\t")[0]
+        else:
+            localeCode = locale.split(" ")[0]
+        if localeCode not in buildsVersionLocales.keys():
+            buildsVersionLocales[localeCode] = [lastVersion]
+        else:
+            buildsVersionLocales[localeCode] += [lastVersion]
+
+    return buildsVersionLocales
+
+
+def fillAuroraVersion(buildsVersionLocales):
+    for localeCode in config.SUPPORTED_AURORA_LOCALES:
+        if localeCode not in buildsVersionLocales.keys():
+            buildsVersionLocales[localeCode] = [config.AURORA_VERSION]
+        else:
+            buildsVersionLocales[localeCode] += [config.AURORA_VERSION]
+
+    return buildsVersionLocales
+
+
+def updateBuilds(product):
```

Comment:
Can you find a better name for this method? It's not really clear to me what it's doing.